odoo/orm/models.py

```diff
@@ -6774,35 +6774,32 @@ def _transient_vacuum(self):
         - the 10 rows that have been created/changed the last 5 minutes will NOT
           be deleted
         """
+        remaining = 0
         if self._transient_max_hours:
             # Age-based expiration
-            self._transient_clean_rows_older_than(self._transient_max_hours * 60 * 60)
+            remaining = self._transient_clean_rows_older_than(self._transient_max_hours * 60 * 60)
 
         if self._transient_max_count:
             # Count-based expiration
-            self._transient_clean_old_rows(self._transient_max_count)
+            remaining = self._transient_clean_old_rows(self._transient_max_count)
+        return self._name, remaining
```

Comment:
returning a str as first tuple element is... odd... I get it you do that so it logs the model name as this ir.autovacuum method is shared by all TransientsModels, but we should at least add a comment